app/services/datastore/application_rehydration.rb

```diff
@@ -40,10 +40,11 @@ def call # rubocop:disable Metrics/MethodLength, Metrics/AbcSize, Metrics/Cyclom
         evidence_last_run_at: evidence_last_run_at,
         evidence_prompts: evidence_prompts,
 
-        # Change in Financial Circumstances
+        # Change in Financial Circumstances specific fields
         pre_cifc_reference_number: pre_cifc_reference_number,
         pre_cifc_maat_id: pre_cifc_maat_id,
         pre_cifc_usn: pre_cifc_usn,
+        pre_cifc_reason: pre_cifc_reason,
```

Comment:
Why have methods for these cifc attributes? rather than just having 
```suggestion
        pre_cifc_reason: parent.pre_cifc_reason,
```